### PR TITLE
Feature/update dependencies

### PR DESCRIPTION
**Pull request by lakpa**

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -74,7 +74,7 @@ class _MyAppState extends State<MyApp> {
   }
 
   _fetchMicrosoftProfile() async {
-    var response = await http.get(this._graphURI, headers: {"Authorization": "Bearer " + this._authToken});
+    var response = await http.get(this._graphURI as Uri, headers: {"Authorization": "Bearer " + this._authToken});
```
Comment (karspv) [inline example/lib/main.dart:77]: Is downcasting alright? maybe we should Parse the url as Uri

File: example/test/widget_test.dart [hunk 2]
```diff
@@ -19,7 +19,7 @@ void main() {
     expect(
       find.byWidgetPredicate(
         (Widget widget) => widget is Text &&
-                           widget.data.startsWith('Running on:'),
+                           widget.data!.startsWith('Running on:'),
```
Comment (karspv) [inline example/test/widget_test.dart:22]: maybe just delete this file :D

**Review by karspv — APPROVED** [review]
Small comment

File: example/lib/main.dart (hunk 1 above)
Comment (lakpa) [reply example/lib/main.dart:77]: It is an example project, that is alright

File: example/test/widget_test.dart (hunk 2 above)
Comment (lakpa) [reply example/test/widget_test.dart:22]: I will delete redundant ones later

**Pull request merged**